app/assets/javascripts/components/EscapeListener.js

```diff
@@ -11,6 +11,14 @@ export default class EscapeListener extends React.Component {
     this.onClick = this.onClick.bind(this);
   }
 
+  componentDidMount() {
```

Comment:
@kevinrobinson didn't mean to have you looking over a half finished request! But this part I did want to check in about since it affects the absence dashboard as well. I wasn't able to get the escape function to work without these event listeners, but presumably it was working at some point. Is there some trick to this component I'm missing?

Reply:
@kevinrobinson Ah, I see the problem there. I'll back out the changes to this one and see if there's a good local solution to the escape listener. Right now it works very inconsistently when the user is interacting with the charts (the student list is fine, though.) I don't think that's critical enough to hold up this PR, though, so I'll make that a separate PR.

Reply:
Sounds good!  Yeah I think improvements to keyboard shortcuts are "nice to have" and so if you can improve it that's cool, but if that seems complicated it's fine to move on to higher-impact work.